arch/riscv64/mcount-support.c

```diff
@@ -0,0 +1,49 @@
+#include <stdlib.h>
+
+#include "libmcount/internal.h"
+#include "utils/filter.h"
+#include "utils/utils.h"
+
+void mcount_arch_get_arg(struct mcount_arg_context *ctx, struct uftrace_arg_spec *spec)
+{
+	/*
+        TODO : This was implemented as a meaningless value because we
+               focused on successful builds with no compilation errors.
+
+               Therefore, it should be modified for future RISC-V 64-bit
+               architectures.
+    */
```

Comment:
At first, do not mix tab and whitespace.

Secondly, I would write this comment as follows.
```c
        /*
         * TODO:
         *   This was implemented as a meaningless value because we focused on
         *   successful builds with no compilation errors.
         *
         *   Therefore, it should be modified for future RISC-V 64-bit
         *   architectures.
         */
```